docs/pages/yaml-reference.md

```diff
@@ -97,6 +93,20 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
   * `strategy`
 
     This field isn't used! TODO(nickbp) remove.
+   
+  * `networks`
+
+    Allows the pod to join any number of virtual networks on the DC/OS cluster, however the only supported virtual network at present is the `dcos` overlay network. To have the pod join the overlay add the following:
+    ```
+    networks:
+      dcos:
+    ```
+    **N.B.** when a pod joins the overlay it has the following effects:
+     * The pod is given it's own IP address from the sub-net of the overlay belonging to the agent the pod is deployed to.
+        * The IP can be retrieved using the DNS `<task_name>.<framework_name>.autoip.dcos.thisdcos.directory` (This DNS will also work for pods on the host network).
+     * The `ports` resource requirements will be ignored (i.e. the agent does not need to have these ports available) because the pod has it's own IP address.
+     * Once this pod is on the overlay, you cannot change the configuration to one where it moves to the host network. This is disallowed because the ports may not be available on the agent that has the rest of the task's reserved resources. 
+     * For more information see the DC/OS Virtual Network [documentation](https://docs.mesosphere.com/1.9/networking/virtual-networks/#virtual-network-service-dns).
 
```

Comment:
- Once the pod is on the overlay, you cannot move it to the host network. This is disallowed because...